### ISSUE TITLE
gi.repository.GLib.GError: gtk-icon-theme-error-quark: Icon 'media-optical' not present in theme Adwaita (0)

### ISSUE THREAD

**Issue by kingosticks** [orontee/argos#157]
I installed the latest deb release on my Ubuntu 22.04LTS system and I get the following traceback when trying to start argos
```
nsteel@NSTEEL3:~$ argos
Traceback (most recent call last):
  File "/usr/bin/argos", line 50, in <module>
    from argos.app import Application
  File "/usr/share/argos/argos/app.py", line 32, in <module>
    from argos.widgets import (
  File "/usr/share/argos/argos/widgets/__init__.py", line 2, in <module>
    from argos.widgets.albumdetailsbox import AlbumDetailsBox
  File "/usr/share/argos/argos/widgets/albumdetailsbox.py", line 30, in <module>
    class AlbumDetailsBox(Gtk.Box):
  File "/usr/share/argos/argos/widgets/albumdetailsbox.py", line 42, in AlbumDetailsBox
    default_album_image = default_image_pixbuf(
  File "/usr/share/argos/argos/widgets/utils.py", line 30, in default_image_pixbuf
    pixbuf = Gtk.IconTheme.get_default().load_icon(icon_name, max_size, 0)
gi.repository.GLib.GError: gtk-icon-theme-error-quark: Icon 'media-optical' not present in theme Adwaita (0)
```
Is this me, or is the app not supported on the current Ubuntu LTS release? 

EDIT: This is actually Ubuntu 22.04 on WSL2... Can I use Flatpak instead here?

**Comment by orontee** [comment]
Thanks for your report! I'll have a look in a couple of days.

**Comment by orontee** [comment]
Sorry, dunno why this is closed!

**Comment by orontee** [comment]
@kingosticks Do you have the `gnome-icon-theme` package installed? If not can you install it and retry?

If it solves the problem, I'll declare this dependency in the deb package.

**Comment by orontee** [comment]
I don't have a Windows box at hand by on a linux box, I created a virtual machine running Ubuntu 22.04 LTS. There I performed:
```shell
$ sudo apt install -yf ~/Téléchargements/argos_1.12.0-1_all.deb 2>&1 /dev/null
$ argos
```
And... everything went fine...

**Comment by kingosticks** [comment]
Sorry, it's my work machine and I've not been back on it yet. I imagine the WSL's Ubuntu doesn't come with the same default packages w.r.t to UI. But it looks like the dependency is real so should be added.

**Comment by orontee** [comment]
I just released 1.13.0 with updated dependencies. I must confess I've not carefully checked that there was no other missing dependency when installing from a bare distribution... Let's reopen this if I was too confident!

**Comment by kingosticks** [comment]
Yep! That did it.
